README.md

```diff
@@ -178,6 +178,8 @@ jobs:
 
 > **Note**: By default not specifying `fromTag` or `toTag` will resolve `toTag` from either the `ref` or alternatively fallback to the latest tag from the git API. `fromTag` is resolved by sorting tags using [semver](https://semver.org/). Check the [configuration](#configuration-specification) for alternatives.
 
+> **Note**: If you behind a corporate HTTP proxy you might set the `HTTPS_PROXY` environment variable to the proxy URL.
```

Comment:
Can you please link to the docs from octokit here - so people can read up on this environment variable.